src/clear/model/modules/has_saving.cr

```diff
@@ -20,13 +20,13 @@ module Clear::Model::HasSaving
 
           if h.any?
             with_triggers(:update) do
-              Clear::SQL.update(self.class.table).set(update_h).where { var("#{self.class.pkey}") == pkey }.execute
+              Clear::SQL.update(@@connection, self.class.table).set(update_h).where { var("#{self.class.pkey}") == pkey }.execute
```

Comment:
Would be better to pass connection as parameter for execute

```crystal
query = SQL.update(table).set(update_h).where{ expr }
query.execute #Connection = "default"
query.execute("secondary") #Connection = secondary
```

The reason is we want to delegate the selection of the connection at the moment it's important. During the SQL composition, we do not care about where it will be executed or even if it will be executed. In the SQL specs, lot of requests are not even executed.

Reply:
Remark: On `fetch` for `select` requests.